lib/appium_lib/common/patch.rb

```diff
@@ -90,7 +90,7 @@ def raw_execute(command, opts = {}, command_hash = nil)
       path_match = path.match /.*\h{8}-?\h{4}-?\h{4}-?\h{4}-?\h{12}/
       path_str   = path.sub(path_match[0], '') unless path_match.nil?
 
-      puts "#{verb} #{path_str}"
+      Appium::Logger.info  "#{verb} #{path_str}"
```

Comment:
This function does nothing other than print to stdout so I don't think a logger makes sense here.